src/main/java/perosnal/spotifymatcher/service/SpotifyApiService.java

```diff
@@ -51,6 +52,7 @@ public void persistUser(String token) {
                                 .country())
                         .build());
 
+        insertFakeUser(user);
```

Comment:
This could be extracted to something like `FakeUserService` that can be toggled on/off (based on an environment variable for example)

Reply:
I'm planning to have fake users generated on every enviroment